levelkdev/swapr-dapp
# fix: optimize on-chain USD price fetching

# Summary

Fixes #1066

Optimize on-chain USD price fetching and prevent laggy UI

  # To Test

1. Open the page `swap`
2. Simulate a trade
3. Open settings modal
- [ ] Verify if changing settings isn't laggy
4. Use the app while simulating the trade
- [ ] Verify if using the app in general isn't laggy
